boards/arm/nucleo_f091rc/doc/nucleof091rc.rst

```diff
@@ -0,0 +1,168 @@
+- ARM* mbed*
+- On-board ST-LINK/V2-1 debugger/programmer with SWD connector:
+    - Selection-mode switch to use the kit as a standalone ST-LINK/V2-1
+- Flexible board power supply:
+    - USB VBUS or external source (3.3V, 5V, 7 - 12V)
+    - Power management access point
+- Three LEDs:
+    - USB communication (LD1), user LED (LD2), power LED (LD3)
+- Two push-buttons: USER and RESET
+- USB re-enumeration capability. Three different interfaces supported on USB:
+    - Virtual COM port
+    - Mass storage
+    - Debug port
+- Support of wide choice of Integrated Development Environments (IDEs) including:
+    - IAR
+    - ARM Keil
+    - GCC-based IDEs
+
+More information about STM32F091RC can be found here:
+       - `STM32F091 reference manual`_
+
+
+Supported Features
+==================
+
+The Zephyr nucleo_f091rc board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | reset and clock control             |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| IWDG      | on-chip    | independent watchdog                |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported in this Zephyr port.
+
+The default configuration can be found in the defconfig file:
+``boards/arm/nucleo_f091rc/nucleo_f091rc_defconfig``
+
+Connections and IOs
+===================
+
+Each of the GPIO pins can be configured by software as output (push-pull or open-drain), as
+input (with or without pull-up or pull-down), or as peripheral alternate function. Most of the
+GPIO pins are shared with digital or analog alternate functions. All GPIOs are high current
+capable except for analog inputs.
+
+Board connectors:
+-----------------
+.. image:: img/nucleo_f091rc_connectors.png
+     :width: 800px
+     :align: center
+     :height: 619px
+     :alt: Nucleo F091RC connectors
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+- UART_1_TX : PA9
+- UART_1_RX : PA10
```

Comment:
Board's DT file configures UART_1 to pins PB6/7. Could you update the mapping here?

Reply:
Done